### PR TITLE
Added a multisquare variant of curtains

### PR DESCRIPTION
**Pull request by Asmageddon**
Also, if anyone has an idea for a better name than "sliding curtain" which mostly just differentiates it from the ordinary curtains, it would be appreciated.

### CODE REVIEW

File: data/json/vehicle_parts.json [hunk 1]
```diff
@@ -1000,6 +1000,12 @@
         "breaks_into": []
     },{
         "type" : "vehicle_part",
+        "copy-from": "v_curtain",
+        "id" : "v_curtain_sliding",
+        "name": "sliding curtain",
+        "flags": ["OPENABLE", "OPENCLOSE_INSIDE", "OPAQUE", "CURTAIN", "MULTISQUARE"]
```
Comment (keyspace) [inline data/json/vehicle_parts.json]: If using `copy-from`, then perhaps `extend` also?..

``` json
"extend": { "flags": [ "MULTISQUARE" ] }
```

Haven't tested.

**Comment by keyspace** [comment]
Note that the rest of the file hasn't been updated to use `copy-from`, `extend`, etc.

-----

IMO PRs like this create unnecesary strain on reviewers. :(

It doesn't:

* close an issue,
*  fix something broken.

It does introduce:

* content that's not tied to the rest (e.g. a stock vehicle),
* a "special case" that makes it harder to port with a script.

Not a big deal, file into complaint bin.

**Comment by Coolthulhu** [comment]
Multitile curtains were rejected sometime ago because of a problem:
They allow closing curtains on a passable tile, thus creating a tile that is opaque but passable. This doesn't happen elsewhere and might confuse AI and vision.

**Comment by Asmageddon** [comment]
I've been working on porting everything to use copy-from and requirements, but I've got no go-ahead on what I'd like to change in an issue I opened, so I'm holding off with that.

Also, it's aisle curtains that can be placed on passable tiles, I don't think you can do that with curtains, although I'm not sure.

File: data/json/vehicle_parts.json (hunk 1 above)
Comment (Asmageddon) [inline data/json/vehicle_parts.json]: Oh, I somehow missed this comment, I'll do that, thanks.

**Pull request closed without merging**